packages/forms/src/documents/pdf/parsing-api.ts

```diff
@@ -36,6 +37,15 @@ const TxInput = z.object({
   page: z.union([z.number(), z.string()]),
 });
 
+const Attachment = z.object({
+  component_type: z.literal('text_input'),
+  id: z.string(),
+  label: z.string(),
+  default_value: z.string(),
+  required: z.boolean(),
+  page: z.union([z.number(), z.string()]),
+});
```

Comment:
@jimmoffet how do you expect the parsing API to handle attachments (if it will)?

Reply:
I think we need attachments for generation but not extraction. For example, if the attachment is an image, it would arrive to the extraction process as an image embedded on a page within the pdf. So, we may eventually want to create an image type here (or any other type of thing that might be embedded in a pdf page), but I don't think we'll an attachment concept here.